package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "0.2.0-beta.243",
+  "version": "1.0.0-beta",
```

Comment:
I think I still prefer it if we have the PR number here.

Reply:
Product asked for this to be the version. I can see if they're okay with us adding the PR number.